src/atoms/pill/buildTheme.ts

```diff
@@ -74,12 +74,24 @@ export const buildPillThemes = (colors: IColorGuide, dimensions: IDimensionGuide
     },
   });
 
+  const squaredPillTheme = mergeThemePartial<IPillTheme>({
```

Comment:
why is this squared? how is it different than the default pill?

Reply:
just it has different padding size
it should not be called squared, as it the text gets long it will be a rectangle